ELA/found/CRSuggestion/ViewController/HWMSuggestionViewController.mm

```diff
@@ -132,14 +132,14 @@ -(void)onTxPublish:(NSNotification*)notice{
     NSMutableDictionary *resultDic = _votingProcessUtil.resultDic;
     
     NSDictionary *param =notice.object;
-    NSLog(@"xxl 943 2 HWMSuggestionViewController onTxPublish %@ 1",param);
+    WYLog(@"xxl 943 2 HWMSuggestionViewController onTxPublish %@ 1",param);
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
         
-        NSLog(@"xxl 943 Code = %@",param[@"Code"]);
-        NSLog(@"xxl 943 _pluginResult = %@",resultDic);
-        NSLog(@"xxl 943 pwd = %@",resultDic[@"pwd"]);
-        NSLog(@"xxl 943 SignTransaction = %@",resultDic[@"SignTransaction"]);
-        NSLog(@"xxl 943 calculateProposalHash = %@",resultDic[@"calculateProposalHash"]);
+        WYLog(@"xxl 943 Code = %@",param[@"Code"]);
+        WYLog(@"xxl 943 _pluginResult = %@",resultDic);
+        WYLog(@"xxl 943 pwd = %@",resultDic[@"pwd"]);
```

Comment:
What does `pwd` mean? Can sensitive information such as passwords be included here?

Reply:
it's removed from updated code